dcpy/lifecycle/ingest/transform.py

```diff
@@ -267,4 +290,8 @@ def preprocess(
 
         for step in compiled_steps:
             df = step(df)
+
+        if output_csv:
```

Comment:
This just seemed like a nice little QOL for troubleshooting - cli flag to output a csv at end of processing

Reply:
I kept having errors at time of parquet generation due to pandas weirdness